vllm/model_executor/models/llama.py

```diff
@@ -198,6 +200,7 @@ def forward(
         kv_cache: KVCache,
         input_metadata: InputMetadata,
         residual: Optional[torch.Tensor],
+        kv_quant_param: List[float],
```

Comment:
`kv_quant_param` is pased by `input_metadata.kv_quant_params[i]`. So there is no need to pass `kv_quant_param`?

Reply:
It is because that `kv_quant_param` is related to the `layer num`. Although we put `kv_quant_params` in `input_metadata`, we still have to get the right `kv_quant_param` for the specific `layer num`. So we capture the `layer num` information at line 270 in llama.py and pass it to the subsequent forward.

Reply:
OK. Thanks for your explanation，